intellst-api/src/Entity/Enterprises.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\EnterprisesRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=EnterprisesRepository::class)
+ */
+class Enterprises
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Users", mappedBy="enterprises")
```

Comment:
Please use `./bin/console doctrine:schema:validate` console command. There is no field `enterprises` in User entity.